wicket-core/src/main/java/org/apache/wicket/core/util/resource/locator/caching/FileSystemResourceStreamReference.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.wicket.core.util.resource.locator.caching;
+
+import java.nio.file.Paths;
+
+import org.apache.wicket.util.resource.PathResourceStream;
+
+/**
+ * A reference which can be used to recreate {@link PathResourceStream}
+ * 
+ * @author Tobias Soloschenko
+ */
+public class FileSystemResourceStreamReference extends AbstractResourceStreamReference
```

Comment:
We should use the same name for all classes - either `FileSystem**` or `Path**`.